roles/jail/tasks/main.yml

```diff
@@ -27,11 +52,13 @@
     mode: "0700"
   when: jail_created|changed
 
-- name: execute bootstrap script
+- name: execute bootstrap script {{ name }}
```

Comment:
yes, it works for me. It helps to identify which jail is processing when starring at the ansible output